frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss

```diff
@@ -9,5 +9,6 @@
 .odc-resource-icon-deployment,
 .odc-resource-icon-daemonset,
 .odc-resource-icon-deploymentconfig {
+  cursor: pointer;
```

Comment:
This should not be a cursor pointer. And nothing should happen when the user clicks on the node name. The only node should be clickable. Can you confirm this with UX?

Reply:
Sorry @vikram-raj saw it now. Only icon has pointer effect i.e `D`, `DC` etc not the node name as we have in other badges for resources. Will check on this with UXD

@openshift/team-devconsole-ux